### PR TITLE
Add namespace tag to KairosDB insert queries

### PR DESCRIPTION
**Pull request by mohabusama**
Add namespace tag to KairosDB insert queries

https://github.com/zalando-zmon/zmon-worker/issues/333

### CODE REVIEW

File: src/main/java/de/zalando/zmon/dataservice/data/KairosDBStore.java [hunk 1]
```diff
@@ -48,7 +31,8 @@
     private final Set<String> entityTagFields;
     // adding alias,account_alias,cluster_alias due to legacy, and should be exclusive anyways
     private final static Set<String> DEFAULT_ENTITY_TAG_FIELDS = new HashSet<>(
-        Arrays.asList("application_id", "application_version", "stack_name", "stack_version", "application","version","account_alias","cluster_alias","alias"));
+        Arrays.asList("application_id", "application_version", "stack_name", "stack_version", "application",
```
Comment (beverage) [inline src/main/java/de/zalando/zmon/dataservice/data/KairosDBStore.java]: If this is constant (looks like), ImmutableSet.of(<varargs>) would be more appropriate here.

Comment (mohabusama) [reply src/main/java/de/zalando/zmon/dataservice/data/KairosDBStore.java]: Done

**Comment by beverage** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Comment by vetinari** [comment]
:+1:

**Comment by beverage** [comment]
:+1:

**Pull request merged**